options.go

```diff
@@ -13,12 +13,23 @@ type Options struct {
 	// Print step definitions found and exit
 	ShowStepDefinitions bool
 
-	// Run scenarios in random order.
+	// Randomize causes scenarios to be run in random order.
 	//
-	// This is especially helpful for detecting situations
-	// where you have state leaking between scenarios, which
-	// can cause flickering or fragile tests.
-	RandomOrder bool
+	// Randomizing scenario order is especially helpful for detecting
+	// situations where you have state leaking between scenarios, which can
+	// cause flickering or fragile tests.
+	Randomize bool
```

Comment:
I have made a little scenario which would work, see https://play.golang.org/p/CUW5cRjNtb that way we would have only one flag and no confusion.